packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.module.css

```diff
@@ -42,18 +42,18 @@
   background-color: var(--color-surface--1);
 }
 
-.connectionTypeInfo > svg {
-  min-width: 40px;
-  min-height: 40px;
-}
-
 .connectionTypeInfo {
   display: flex;
   gap: var(--space--16);
```

Comment:
You can bump the gap to `--space--24` to take the comment below about the icon size into account.